frontend/src/settings/display/ExampleDisplay.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Button } from '@mui/material'
+import { ThemeProvider } from '@mui/material/styles'
+import { patchCurrentUserSettings, useGetCurrentUserSettings } from 'actions/user'
+import { ThemeKey } from 'src/theme'
+
+interface ExampleDisplayProps {
+  theme: ThemeKey
+}
+export default function ExampleDisplay({ theme }: ExampleDisplayProps) {
+  const { mutateUserSettings } = useGetCurrentUserSettings()
+
+  async function setTheme() {
+    const updatedSettings = await patchCurrentUserSettings({ theme: theme.key })
+    if (updatedSettings.status === 200) {
+      mutateUserSettings()
+    }
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
This should be able to be removed as it's already included in `_app.tsx`

Reply:
Each example display has its own theme (so you can see what the colours are!).

![image](https://github.com/gchq/Bailo/assets/89992537/b7e2e804-3831-446e-a248-d1a96e5cc6e4)

Ideally I'll improve it so they're not just buttons